### PR TITLE
Added tests for views/start.vue

### LINKED ISSUE #464: Add unit tests for views/start
## ✨ Feature request
I want to know ~~if~~ when I break stuff.

### 📜 Acceptance Criteria
- [ ]  There should be 50% code coverage on this component

### 💡 Additional context
https://lmiller1990.github.io/vue-testing-handbook/

### PR DESCRIPTION
**Pull request by te4-david-jensen**
## ⚙️ What issue did you implement or fix?
Fixes #464 

## 📜 Summary
- Implemented Unit tests for views/start.vue

## 📝 Type of change
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Increase test coverage

### CODE REVIEW

**Comment by te4-david-jensen** [comment]
Adding 'help wanted' as I can't make heads nor tails of how to fix the build errors in the integration tests.

**Comment by te4-david-jensen** [comment]
the typeIssue has now been resolved

**Comment by itggot-linus-styren** [comment]
@te4-david-jensen Can "DO NOT MERGE YET" be removed now?

**Comment by te4-david-jensen** [comment]
> @te4-david-jensen Can "DO NOT MERGE YET" be removed now?

Forgot about that; removed it.

**Review by Te4-Jacob-sandstrom — APPROVED** [review]
Everything looks good

File: frontend/src/views/Start.vue [hunk 1]
```diff
@@ -87,24 +87,24 @@
   LoanListComponent in order to render the loans that soon are to expire
 -->
 <script lang="ts">
-import { SetupContext } from '@vue/composition-api';
+import { defineComponent, SetupContext } from '@vue/composition-api';
```
Comment (TE4-Filip-Liljenberg) [inline frontend/src/views/Start.vue:90]: import ref

File: frontend/src/views/Start.vue [hunk 2]
```diff
@@ -87,24 +87,24 @@
...
 
-export default ({
-  name: '',
+export default defineComponent({
+  name: 'StartView',
   components: {
     LoanListComponent
   },
   setup(_ : object, { root } : SetupContext): object {
     const size: string = 'lg';
```
Comment (TE4-Filip-Liljenberg) [inline frontend/src/views/Start.vue:104]: Do, const size = ref('lg')

File: frontend/src/views/Start.vue [hunk 3]
```diff
@@ -87,24 +87,24 @@
...
     const primary: string = 'primary';
     const loading: boolean = true;
```
Comment (TE4-Filip-Liljenberg) [inline frontend/src/views/Start.vue:106]: const loading = ref(true);

File: frontend/src/views/Start.vue [hunk 4]
```diff
@@ -87,24 +87,24 @@
...
-export default ({
-  name: '',
+export default defineComponent({
+  name: 'StartView',
   components: {
     LoanListComponent
   },
   setup(_ : object, { root } : SetupContext): object {
     const size: string = 'lg';
     const primary: string = 'primary';
```
Comment (TE4-Filip-Liljenberg) [inline frontend/src/views/Start.vue:105]: const primary = ref('primary')

File: frontend/src/views/Start.vue [hunk 5]
```diff
@@ -87,24 +87,24 @@
...
     const loading: boolean = true;
-    const usersModule: VuexModule = UsersModule;
+    const usersModule: UserModuleType = UsersModule;
```
Comment (TE4-Filip-Liljenberg) [inline frontend/src/views/Start.vue:107]: const usersModule = ref(UserModuleType)

**Review by TE4-Filip-Liljenberg — CHANGES_REQUESTED** [review]

File: frontend/src/views/Start.vue (hunk 5 above)
Comment (itggot-linus-styren) [reply frontend/src/views/Start.vue:107]: I suggest to remove this line completely and put UsersModule in return of setup directly

Comment (te4-david-jensen) [reply frontend/src/views/Start.vue]: How would I go about returning UserModule directly?

**Review by TE4-Filip-Liljenberg — APPROVED** [review]
Good job!

**Pull request merged**